app.js

```diff
@@ -1,10 +1,27 @@
+/*jslint node: true */  
```

Comment:
You no longer need /*jslint node: true */ , your .jshintrc file is taking care of that.